app/src/main/res/layout/fragment_profile.xml

```diff
@@ -28,19 +28,21 @@
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
         android:layout_centerHorizontal="true"
-        android:layout_marginBottom="530dp"
+        android:layout_marginBottom="582dp"
```

Comment:
how was this value calculated?  What would happen in landscape mode or on small screens?

Reply:
It was done based on my screen but I have been able to adjust to work on either big or small screens